script/chart_sync_utils.sh

```diff
@@ -71,6 +71,7 @@ replaceImage() {
 updateRepo() {
     local targetRepo=${1:?}
     local targetTag=${2:?}
+    local targetTagWithoutV=${targetTag/v/}
```

Comment:
A bit pedantic, but you probably want to strip from the front of the var, rather than replacing anywhere in the var?
```
$ VERSIONTEST=v1.2.3 && echo ${VERSIONTEST#v}
1.2.3

$ VERSIONTEST=1.2.3v && echo ${VERSIONTEST#v}
1.2.3v

$ VERSIONTEST=1.2.3v && echo ${VERSIONTEST/v/}
1.2.3
```